### ISSUE TITLE
[Feature]: Template and Entity card support for icon tap action

### ISSUE THREAD

**Issue by samuolis** [piitaya/lovelace-mushroom#814]
### Is your feature request related to a problem?

No

### Describe the solution you'd like

In 2022.11 Home assistant introduces tile card, it would be very nice to have same functionality of icon tap action.

### Describe alternatives you've considered

Use tile card instead

**Comment by cgilis** [comment]
+1

**Comment by sstrang** [comment]
+1. I really liked this idea from the new tile card, but didn't like that it is much less configurable than the entity and template cards from this project. I would love to be able to have the option to tap a template card's icon to toggle the entity, and tap the rest of the card to bring up More Info.

**Comment by PolestarWx** [comment]
Great idea !
Also atwork backgorund for media card and perfect :DDD

**Comment by awashingmachine** [comment]
It would be a great addition! 
I like the cards to be displayed horizontally and from time to time I accidentally turn on lights, scenes or scripts. (I don't like to double tap the card to toggle it)

**Comment by DIYtechie** [comment]
I very much agree. Tap_icon_action should be anabled for all cards with icons.

 The new tile cards cannot show secondary template info, so not a real alternative.

**Comment by DIYtechie** [comment]
Please consider adding tap_icon_action. This would provide a much better interface for mushroom based room cards than the current hold action.

**Comment by DIYtechie** [comment]
@piitaya What are your thoughts on this? For me it would be a big deal and I'm sure that many Mushroom users would use it, since it is much simpler than "hold" or "double press". I find the hold action is a bit hit and miss and double press adds a small but noticeable delay.

**Comment by PolestarWx** [comment]
Yes great idea ! tile card from HA provide this function but no template possible

**Comment by DIYtechie** [comment]
I have still not found a good alternative to this. @piitaya, is it being considered? 

Would happily make an in advance donation to support the development of this feature (without being able to pay the hourly rate you deserve).

**Comment by broyuken** [comment]
I would like to request this feature as well. The tile card does not let me do certain things like collapse the controls when off that mushroom allows, but having the tap icon action is essential

**Comment by Nickduino** [comment]
Agreed, it would be very useful on the light (or cover) card:
- taping the icon will toggle on/off (or open/closed);
- taping the card will open the "More" popup with the ability to use the slider, change color, etc.
